### DISCUSSION TITLE
button highlight for skillchains

### DISCUSSION THREAD

**Discussion by johnnyrags** [Technyze/XIVHotbar2#4] [Ideas]
I understand that this would be a fair bit of work to parse all of the skillchain combination information, but would a feature request potentially be workable to see about having buttons highlight with either the FFXIV "ants crawling" edge animation or some kind of opacity color overlay on the button to denote that using said spell/skill/ja would chain?

I feel like this would be a pretty awesome addition, more so for players that are not yet well learned enough to know all of the combinations from their class with others.

Thanks Technyze for making this, looking forward to your continued work

**Comment by Technyze** [comment]
Thats a really great idea and in fact that is similar to something I kind of wanted to do to show when a spell could be upgraded. Determined this was a large undertaking and decided instead to put a glowy scroll icon overlay instead. If I do something like this is will probably have to be far in the future or if I ever decide to completely remake the addon.
